resource/out

```diff
@@ -28,6 +28,8 @@ name=$(jq -r '.source.name' < "$payload")
 bucket=$(jq -r '.source.bucket // ""' < "$payload")
 # commit ref to release
 repository=$(jq -r '.params.repository // ""' < "$payload")
+# Whether the deployment includes a docker image
+has_image=$(jq -r '.source.has_image' < "$payload")
```

Comment:
```suggestion
has_image=$(jq -r '.source.has_image // "false"' < "$payload")
```

Just to maintain it backwards compatible?  Or when we compare to `== "false"` we do an `else` instead of another `if`